api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -141,6 +141,22 @@ service ConfigMgmt {
     option (chef.automate.api.iam.policy).action = "infra:nodes:list";
   };
 
+  /*
+  GetMissingNodeDurationCounts
+  Returns a count of missing nodes for the provided durations.
+  Authorization Action:
```

Comment:
can you include a comment about valid duration formats here?

Reply:
Thank you for pointing that out. I updated the docs also. They look like below. 

![image](https://user-images.githubusercontent.com/1679247/78608349-9fde1700-7815-11ea-8782-199ba1ec171b.png)